### DISCUSSION TITLE
Define additional fields for prisma connection object

### DISCUSSION THREAD

**Discussion by julioxavierr** [hayes/pothos#1281] [Q&A] [answered]
The documentation includes a [section demonstrating how to extend edges](https://pothos-graphql.dev/docs/plugins/prisma/connections#extending-connection-edges). However, is there a way to extend the connection object?

For example:

```graphql
query {
  users {
    totalCount
    totalActive # additional field
  }
}
```

I understand it's possible to create a connection object and assign that as the second argument of `prismaConnection`, but I'm not sure about the potential downsides of that approach.

**Comment by hayes** [comment, marked as answer]
There's no downside to creating the connection object yourself.  Internally, it will just call `builder.connectionObject({...})` for you if you don't provide your own connection object.

You can also just define the fields inline in the connection options:

```ts
t.prismaConnection(
  {
    type: 'User',
    cursor: 'id',
    resolve: (query) => db.game.findMany({ ...query  }),
  },
  {
    fields: (t) => ({
      totalActive: t.int({
        resolve: () => 5,
      }),
    }),
  },
)
```
